integration-tests/ahm/src/tests.rs

```diff
@@ -743,18 +774,23 @@ async fn scheduled_migration_works() {
 		warm_up_end = start + 3;
 
 		// Fellowship Origin
+		#[cfg(not(feature = "kusama"))]
```

Comment:
```suggestion
		#[cfg(feature = "polkadot")]
```
We can make it explicit now since Paseo is gone.